### ISSUE TITLE
ImageMagick convert Illegal Instruction 

### ISSUE THREAD

**Issue by HimbeersaftLP** [ish-app/ish#1417]
Trying to convert this image from jpg to png (but the same error occurs with other images and other convert options)
![penguins-5602408_640](https://user-images.githubusercontent.com/16354747/119222715-00054a80-baf6-11eb-8959-873d57cf4ac7.jpg)

```
iPad:~# convert penguins-5602408_640.jpg penguins-5602408_640.png
Illegal instruction
iPad:~# dmesg
19 illegal instruction at 0xf760b0b9: 66 0f d5 02 66 0f d5 4a

**Comment by saagarjha** [comment]
`pmullw`

**Comment by banderlog** [comment]
same problem with any image, any imagemagick command or caca utils

- ish version: 1.2.3
- phone: iPhone SE
    - model number: MX9U2FS/A
    - ios version: 15.6

**Comment by banderlog** [comment]
from [discord official group](https://discord.com/channels/508839261924229141/510973900209913857/1006852395298467850):

>This is unfortunately not going to work until the missing instruction is implemented, as saagar mentioned in the GitHub thread it's pmullw

**Comment by Mnpn** [comment]
Hi! #1871 has been merged, implementing `pmullw`. The command now hits the unimplemented instruction `punpcklwd` (`66 0f 61 e3 66 0f 69 eb`), which would be resolved by #1872.

**Comment by Mnpn** [comment]
The PR mentioned above has been merged. ImageMagick is still hitting an unimplemented instruction though, this time `packuswb`.

**Comment by banderlog** [comment]
> The PR mentioned above has been merged. ImageMagick is still hitting an unimplemented instruction though, this time `packuswb`.

might you please create a relevant "add puckuswb" issue?

**Comment by jason-conway** [comment]
Whoops I thought I had `packuswb` included in #1872, my apologies.

`convert` ends up hitting `pinsrw` which can't be implemented quite yet

**Comment by banderlog** [comment]
>convert ends up hitting pinsrw which can't be implemented quite yet

that's sad😔

**Comment by jason-conway** [comment]
Should be resolved by #2036

<img width="404" alt="ish_screen" src="https://user-images.githubusercontent.com/52170171/217409464-7d14e2f5-1d9a-4cfc-8bdc-390fae6610c6.png">

**Comment by Mnpn** [comment]
Hi! Good news - as was mentioned above, this has been resolved by @jason-conway, and the fix is shipping in iSH 1.3 which is now available :) Please give it a try, and please let us know if you're still running into issues with ImageMagick. Thank you!

**Comment by HimbeersaftLP** [comment]
Works great, thanks!
